fmriprep/cli/run.py

```diff
@@ -274,6 +276,26 @@ def get_parser():
     g_other.add_argument('--sloppy', action='store_true', default=False,
                          help='Use low-quality tools for speed - TESTING ONLY')
 
+    currentv = Version(__version__)
+    latest = check_latest()
+
+    if latest is not None and currentv < latest:
+        print("""\
+WARNING: The current version of fMRIPrep (%s) is outdated.
+Please consider upgrading to the latest version %s.
+Before upgrading, please consider that mixing fMRIPrep versions
+within a single study is strongly discouraged.""" % (
+            __version__, latest), file=sys.stderr)
```

Comment:
This is sending some mixed messages, by making using an old version a WARNING but then also saying not to mix versions.

What about making it more informational:

```
Newer version of fMRIPrep available: {latest}
https://fmriprep.readthedocs.io/en/latest/faq.html#upgrading
```

And we could add a note on upgrading that talks about whether to upgrade, which should be in our docs anyway.